roadmap/README.md

```diff
@@ -45,6 +45,7 @@ _Status description:_
 | ✏️️| Add SDKs for more languages (Python, PHP, Rust, etc) |   |
 | ✏️️| Add more samples  |   |
 | ✏️️| Enforce SemVer `version`  |   |
+| ✏️️| Add `dataOutputSchema`  |   |
```

Comment:
so which one is it? dataOutputSchema as requested or inputDataSchema as in the change you made to workflow.json :)

Reply:
If I revert to dataInputSchema, I guess thats fine isnt it?

Reply:
imho yes, but depends if you and others want to do that change. either way after you guys decide 
please make sure that its reflected in readme.md with the same names you use in workflow schema

Reply:
@tsurdilo   I revert to the original form, dataInputSchema and dataOutputSchema so it is aligned with readme roapmap.